builders/build/eth-api/dev-env/foundry.md

```diff
@@ -461,4 +460,33 @@ npm run test
 
 Finally, while not necessary, it could be worthwhile to move all JavaScript scripts from the `scripts` folder into Foundry's `script` folder and delete the `scripts` folder so that you don't have two folders that serve the same purpose.
 
+## Deploying via Solidity Scripting {: #deploying-via-solidity-scripting }  
```

Comment:
I think this might go well under the deployment section. It feels like an after thought down here. But instead of just moving this up, I do think the deploy section could be modified slightly though as to present the two options as equals